app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabs.kt

```diff
@@ -0,0 +1,294 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import mozilla.components.browser.state.state.ContentState
+import mozilla.components.browser.state.state.TabSessionState
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.PrimaryText
+import org.mozilla.fenix.compose.SecondaryText
+import org.mozilla.fenix.compose.list.ExpandableListHeader
+import org.mozilla.fenix.compose.list.FaviconListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+private val ROUNDED_CORNER_SHAPE = RoundedCornerShape(8.dp)
+
+/**
+ * Top-level Composable for displaying an expandable section of Inactive Tabs
+ *
+ * @param inactiveTabs List of [InactiveTabListItem] to display.
+ * @param expanded Boolean to determine whether to show the inactive tabs section expanded or collapsed.
+ * @param showAutoCloseDialog Boolean to determine whether to show the auto close inactive tabs dialog.
+ * @param onHeaderClick Called when the user clicks on the inactive tabs section header.
+ * @param onDeleteAllButtonClick Called when the user clicks on the delete all inactive tabs button.
+ * @param onAutoCloseDismissClick Called when the user clicks on the auto close dialog's dismiss button.
+ * @param onEnableAutoCloseClick Called when the user clicks on the auto close dialog's enable button.
+ * @param onTabClick Called when the user clicks on a specific inactive tab.
+ * @param onTabCloseClick Called when the user clicks on a specific inactive tab's close button.
+ */
+@Composable
+@Suppress("LongParameterList")
+fun InactiveTabsList(
+    inactiveTabs: List<InactiveTabListItem>,
+    expanded: Boolean,
+    showAutoCloseDialog: Boolean,
+    onHeaderClick: () -> Unit,
+    onDeleteAllButtonClick: () -> Unit,
+    onAutoCloseDismissClick: () -> Unit,
+    onEnableAutoCloseClick: () -> Unit,
+    onTabClick: (InactiveTabListItem) -> Unit,
+    onTabCloseClick: (InactiveTabListItem) -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp, vertical = 12.dp)
+            .clip(ROUNDED_CORNER_SHAPE)
+            .background(FirefoxTheme.colors.layer2)
+            .border(
+                width = 2.dp,
+                color = FirefoxTheme.colors.borderPrimary,
+                shape = ROUNDED_CORNER_SHAPE,
+            )
+            .padding(vertical = 8.dp)
+    ) {
+        InactiveTabsHeader(
```

Comment:
The ripple effect I previously mention should still look like:
https://user-images.githubusercontent.com/1190888/168111528-0c235f15-8a26-4411-becc-91a560b7ac42.mov

So, I am expecting the whole "Column" or "Card" to be clickable in this case in order to get the same effect. You might want to look at the Collections composable, but I don't know how helpful that will be since that one is currently rendered view by view.